authalligator_client/entities.py

```diff
@@ -165,6 +165,27 @@ class Account(BaseAAEntity):
     )  # type: Optional[datetime.datetime]
 
 
+@attr.attrs(frozen=True)
+class DeleteOtherAccountKeysPayload(BaseAAEntity):
```

Comment:
This should eventually be a common base class for the deletion operations.

We could also just have an empty class -- if that works with attrs. There's not much point with doing all the rewriting of `_`, which useless anyways.

Reply:
I'll test it, but I'm not sure that you can initialize an empty attrs class (particularly with the entity converter). I know that mypy complained about the empty class edge case, but I ignored it since the base is never initialized in practice.

Reply:
Looks like it works, so I've dropped the field.